lib/auth/auth.go

```diff
@@ -1610,15 +1643,24 @@ func (a *Server) NewWebSession(req types.NewWebSessionRequest) (services.WebSess
 		return nil, trace.Wrap(err)
 	}
 	bearerTokenTTL := utils.MinTTL(sessionTTL, BearerTokenTTL)
-	return services.NewWebSession(token, services.KindWebSession, services.KindWebSession, services.WebSessionSpecV2{
+
+	currTime := a.clock.Now()
+	if !req.LoginTime.IsZero() {
+		currTime = req.LoginTime
+	}
```

Comment:
Nit: Calling this variable `currTime` is confusing since it isn't always the current time.  Maybe call it something like `startTime` or `fromTime`?

Reply:
renamed to `startTime`